keras_retinanet/bin/train.py

```diff
@@ -352,6 +354,25 @@ def check_args(parsed_args):
 
     return parsed_args
 
+def optimizers(args):
```

Comment:
Prefer to name it `create_optimizer`, so that it is in line with the other names.